.github/workflows/go.yml

```diff
@@ -85,7 +84,7 @@ jobs:
               run: make test
 
     build-amazonlinux-latest:
-        if: github.repository_owner == 'aws'
+#        if: github.repository_owner == 'aws'
```

Comment:
Why was this required? Also, if we want to check this in, we probably shouldn't have commented out code.

Reply:
We didn't run CI on amazonlinux, because we dont have that runner on our repo. But flow worked, so I commented out that condition and fixed CI.
Not sure whether GHA uses amazonlinux or ubuntu there...
Waiting for a response from amazon team

Reply:
I want to keep this as is until all CI run on upstream. I'll remove if it passes or revert these files if it fails.

Reply:
I suppose that's okay. I think you could also make a draft PR upstream and test CI that way, but I don't care as long as it gets tested upstream eventually.